tests/unit-tests/test-class-usage-tracking.php

```diff
@@ -116,10 +144,10 @@ public function testAjaxRequestDisableTracking() {
 			$this->assertEquals( array(), $wp_die_args['args'], 'wp_die call has no non-success status' );
 		}
 
-		// Refresh settings
-		Sensei()->settings->get_settings();
+		// Refresh flag
+		$this->refreshTrackingEnabledFlag();
```

Comment:
It has to be done after the setting is set, and before we test for it. It's because of the way Sensei's Setting class works. This call reloads the settings from the DB. Otherwise it just uses the in-memory version, which isn't necessarily up to date.